bin/npg_deletable_dr_runs

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use FindBin qw($Bin);
+use lib ( -d "$Bin/../lib/perl5" ? "$Bin/../lib/perl5" : "$Bin/../lib" );
+use Carp;
+use English qw(-no_match_vars);
+use Getopt::Long;
+use Pod::Usage;
+
+use npg_tracking::Schema;
+
+our $VERSION = '0';
+
+my $help;
+my $verbose;
+GetOptions ('verbose' => \$verbose,
+            'help|?'   => \$help)
+  or croak("Error in command line arguments\n");
+
+pod2usage(1) if $help;
+
+#my $rs=npg_tracking::Schema->connect()->resultset("Run")->search();
+my $rs=npg_tracking::Schema->connect()->resultset(q(RunStatusDict))
+    ->search({description=>[q[run cancelled],q[data discarded],q[qc complete]]})
+    ->related_resultset(q(run_statuses))->search({q(run_statuses.iscurrent)=>1})
+    ->related_resultset("run")->search();
+my $total = $rs->count;
+my $run;
+my $found = 0;
+while (my $run=$rs->next()) {
+    my $status = $run->current_run_status_description || "";
+    my $tag = $run->is_tag_set("staging");
+    my $run_folder = $run->folder_name || "";
+    my $path_glob = $run->folder_path_glob || "";
+    if ($run_folder and !$tag and $path_glob=~/esa/) {
```

Comment:
```suggestion
    if ($run_folder and !$tag ) {
```
as condition checked in DBIC search above (taking query results down from 10s of thousands to hundreds)